molgenis-metadata-manager/src/main/frontend/src/store/utils/flow.types.js

```diff
@@ -0,0 +1,98 @@
+// @flow
```

Comment:
why is this file located in store/util ? I would think it would be top level in frondend